datasets/chalmers-ccic.yaml

```diff
@@ -0,0 +1,50 @@
+Name: Chalmers Cloud Ice Climatology
+Description: The Chalmers Cloud Ice Climatology (CCIC) is a novel, deep-learning-based climate record of ice-particle concentrations in the atmosphere. CCIC results are available at high spatial and temporal resolution (0.07° / 3 h from 1983, 0.036° / 30 min from 2000) and thus ideally suited for evaluating high-resolution weather and climate models or studying individual weather systems.
+Documentation: https://ccic.readthedocs.io
+Contact: https://github.com/see-geo/ccic
+ManagedBy: "[Geoscience and Remote Sensing at Chalmers University of Technology](https://www.chalmers.se/en/departments/see/research/geo)"
+UpdateFrequency: Daily or quarterly, depending on the product
+Tags:
+  - atmosphere
+  - aws-pds
+  - climate
+  - deep learning
+  - environmental
+  - exploration
+  - geophysics
+  - geoscience
+  - geospatial
+  - global
+  - ice
+  - planetary
+  - satellite imagery
+  - zarr
+License: "[CC BY 4.0](https://creativecommons.org/licenses/by/4.0/)"
+Citation: "CCIC was accessed on [DATE] at registry.opendata.aws/chalmers-ccic"
```

Comment:
I just made a PR on the ccic main repo that adds a "Citing CCIC" section to the documentation. I'd suggest refering to it here, so then we can update this once the climate record paper is out. The link will be 
https://ccic.readthedocs.io/en/latest/intro.html#applications once the PR is merged and the docs are built.

Reply:
So I suggest something like: Please refer to [https://ccic.readthedocs.io/en/latest/intro.html#application](https://ccic.readthedocs.io/en/latest/intro.html#applications) for instructions on how to cite the CCIC data.